index.bs

```diff
@@ -364,58 +369,62 @@ to ease the use of Suborigins in legacy applications are also presented. See
 ## Representation of Suborigins ## {#representation}
 
 At an abstract level, a suborigin consists of the <a>scheme</a>, <a>host</a>,
-and <a>port</a> of a traditional origin, plus a <a>suborigin namespace</a>.
+and <a>port</a> of a physical (traditional) origin, plus a <a>suborigin namespace</a>.
```

Comment:
I think you should remove the "(traditional)" parentheticals. I added #17 so that we eventually define 'physical origin' in the spec, and then we can make these all references to those.